graph-builder/src/graph.rs

```diff
@@ -31,6 +31,11 @@ lazy_static! {
         "Number of releases in the final graph, after processing"
     )
     .unwrap();
+    static ref GRAPH_LAST_SUCCESSFUL_REFRESH: IntGauge = IntGauge::new(
```

Comment:
maybe `GRAPH_UPSTREAM_LAST_SUCCESSFUL_REFRESH_TIMESTAMP` for consistency?

Reply:
Similar no, see below.